Dockerfile

```diff
@@ -8,7 +8,7 @@ ARG mattermost_version=7.1.4
 # Update ca-certificates before running git clone to ensure certs are up to
 # date.
 RUN apt-get -y update && \
-    apt-get -y dist-upgrade && \
+    apt-get -y upgrade && \
```

Comment:
May I ask the reason for the change here?

Reply:
hadolint was complaining about the usage of `dist-upgrade`. I can't think of a reason we would need `dist-upgrade` instead of `upgrade`.

Reply:
Gotcha, thanks!